cluster/ci/monitoring/build_cop.md

```diff
@@ -0,0 +1,14 @@
+# build-cop dashboard in prow-monitoring
+
+The [build-cop dashboard](https://grafana-prow-monitoring.svc.ci.openshift.org/d/6829209d59479d48073d09725ce807fa/build-cop-dashboard?orgId=1) in [prow-monitoring](README.md) is an alternative tool which shows the sucessful rate for varous types of prow jobs in build-cop reports. The data in metrics are obtained by scaping the related component in prow. Every authenticated user of our CI cluster has the access to dashboard.
+
+[Prow deck](https://prow.svc.ci.openshift.org/) has been used for the similar purpose. Eg, deck page shows `Success rate over time: 3h: 78%, 12h: 81%, 48h: 77%` for job with name `*-master-e2e-aws`.
+
+The first panel `Job Success Rates for pre-defined job names` in the dashboard shows its success rate (and other jobs related to build-cop reports) at any time point with a time-range, by default, of the last 24 hours.
+
+Our target that _the pass rate of *-master-e2e-aws jobs over the last day should be 75% or higher_ can be satisfied if in the panel, the lowest point during the last 24 hours for `*-master-e2e-aws` is above 75%.
```

Comment:
Can we add horizontal rules to the graph?

Reply:
Respond all except this.
Testing with staging now.

Reply:
About Y-axis label: Will see how [this PR](https://github.com/grafana/grafonnet-lib/pull/122) goes
About the horizontal lines (in our case: 75% and 97% as targets): [other people want it too](https://github.com/grafana/grafana/issues/9085)

Reply:
One improvement I pushed: use percentage as format for the numbers on Y-Axis.

Reply:
Yeah, as per your tests adding the horizontal lines isn't super useful